administration/intro.adoc

```diff
@@ -262,11 +263,48 @@ ____________________________________
 Note: Updates are not yet supported.
 ____________________________________
 
-When using the stable deployment flow, usually it is sufficient to re-apply the manifests for performing a
+In both cases usually it is sufficient to re-apply the manifests for performing a
```

Comment:
This is WIP for the operator. Updating the operator itself by just applying the new manifest might work, but it doesn't know yet how to deal with an updated CR.

Reply:
How do you prefer to pronounce it? Currently, from pronouncing it does not give any guarantees, so I think it is good enough.

Reply:
mh, not sure. @fabiand wdyt?

Reply:
Updates are not supported.

Without guarantees a user can try to re-apply a manifest.
I'd definitely avoid to say "it is sufficient"

Reply:
Please change this, the we are good  @cynepco3hahue

Reply:
done